### DISCUSSION TITLE
也许是最后一次换主题了 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#9] [Announcements]
# 也许是最后一次换主题了 - 没有秃头基因的燊的博客

换到volantis之后，大概是不会再换了 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/30460/

**Comment by STEVE-1936550490** [comment]
博主怎么不更新了呀？:star_struck:

**Reply by yzs020220** [reply]
平淡的生活没有内容可以更新:expressionless:

**Comment by Merinthae** [comment]
看了你的心路历程，真的觉得很奇妙，也很受启发，恰好我现在正用的是fluid。

**Reply by yzs020220** [reply]
Fluid也很好用的，可以先动手用用博客再搞花里胡哨的，也挺好玩的。

**Comment by Archerui** [comment]
大佬想请教一下你是怎么配置数学公式渲染的部分的呀？我用的也是volantis主题，但我试了好多种方法都不太顺利。

**Reply by yzs020220** [reply]
一般都使用更换hexo渲染器markdown-it-plus，然后在front-matter中使用plugins: katex，好像基本就ok了？因为时间比较久，加上我是原本的主题就搞定了公式渲染部分的，所以可能给不了特别好的建议，你可以参照下这篇博客，我自己很多东西都是看这篇博客加的[链接](https://blog.dearxuan.com/2023/01/07/Volantis%E9%AD%94%E6%94%B9%E6%95%99%E7%A8%8B/#katex%E5%85%AC%E5%BC%8F%E6%B8%B2%E6%9F%93%E9%80%9A%E7%94%A8)

**Reply by Archerui** [reply]
好的，谢谢大佬！！！
